doc/cygnus-ngsi/installation_and_administration_guide/name_mappings.md

```diff
@@ -0,0 +1,75 @@
+
+* The original headers sent by `NGSIRestHandler`, i.e. `fiware-service`, `fiware-servicepath`, `fiware-correlator` and `transaction-id`.
+* New headers regarding the mapped FIWARE service and FIWARE service path, i.e. `mapped-fiware-service` and `mapped-fiware-service-path`.
+* The original notification already parsed as a `NotifyContextRequest` object.
+* The mapped version of the original notification as a `NotifyContextRequest` object.
+
+Please observe no raw bytes about the body are sent.
+
+Whenever a sink takes one of these `NGSIEvent`'s, it is only a matter of deciding if such a sink enables the name mappings (`enable_name_mappings` parameter) or not. If name mappings are enabled, then the already parsed `NotifyContextRequest`, mapped version, is used. If not, then the original version is used.
+
+##Important notes
+Please observe the name mappings definition is global to all the sinks, at `NGSIRestHandler`, but then the application is local to the sink, depending on the `enable_name_mappings` parameter. Thus, if any of yur sinks is going to take advantage of the mappings, simply leave blank the mappings configuration file in the REST handler. That will avoid unncessary interception and iterations on the mappings.
+
+Additionally, please observe if any of the original names is not present, then the mapping affects all the names of that type. For instance, if `originalService` is not present in the fist mapping, then the mapping affects all the FIWARE services.
+
+[Top](#top)
+
+##Name mappings vs. grouping rules
+As seen, the name mappings feature is quite similar to the already existent grouping rules. Both of them are Flume interceptors and bot h of them allow chaning certain notified name elements. Thus, which are the differences? Mainly:
+
+| Name mappings | Grouping rules |
+|---|---|
+|Allow changing the notified <b>FIWARE service</b>, <b>FIWARE service path</b>, <b>entity IDs</b>, <b>entity types</b>, <b>attribute names</b> and <b>attribute types</b>.|Allow changing the notified <b>FIWARE service path</b> and the <b>concatenation of entity ID and entity type</b> (this is called the <i>destination</i>).|
+|Plain Flume `Event`'s are intercepted, and `NGSIEvent`'s are put into the channel. Because the intercetor needs to parse the original notidfication, a `NGSIEvent` already contains the original notification parsed, and the mapped version of the original notification, <b>freeing the sinks to parse the notification</b>.|Plain Flume `Event`'s are intercepted, and plain `Event`'s are put into the channel. Thus, <b>the sinks must parse the notification</b>, despite the grouping interceptor already parsed it`.|
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1198/commits/3f5e44ab8c66588e5fa37a684e128ebd50852f79